tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "declaration": false,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "lib": ["es6", "dom"],
+    "lib": ["es6", "dom", "es2018.promise"],
```

Comment:
Do we need `es2018.promise` for the Monaco Editor?

Reply:
For some reason, when we remove the `react-refresh` library, we need this now. It allows us to use `finally`.

Reply:
Yes. Finally is not part of es2015 promises